src/app/services/actions/permissions-action-creator.ts

```diff
@@ -115,7 +115,7 @@ export function consentToScopes(scopes: string[]): Function {
       dispatch(
         setQueryResponseStatus({
           statusText: translateMessage('Scope consent failed'),
-          status: errorCode,
+          status: errorCode + ' ' + translateMessage('Tip') + ': ' + translateMessage('Access to permission denied'),
```

Comment:
**question:** Is this the only situation that consenting to permissions fails?

Reply:
They are two situations. The first one is an authentication error (access_denied) when a user is signing in to GE for the first time : If the user's tenant admin has restricted consent to client applications, they'll get the access denied error (Authentication failed: access_denied)
![image](https://user-images.githubusercontent.com/45680252/129780038-5232cc9a-0335-4a94-a79d-974be5a81b0e.png)

The second one is when a user is signed in to GE but they cannot consent to certain permissions because the tenant admin has blocked that. For this case, the user gets a 'Scope consent failed: access_denied
![image](https://user-images.githubusercontent.com/45680252/129777182-fbd38147-3e8e-4a22-b639-06ea3fd0801b.png)

Is there a different scenario I have left out?